generator/src/main.rs

```diff
@@ -1621,7 +1621,7 @@ impl Parser {
             out.has_pointer |= member.ptr_depth != 0 || self.handles.contains(&member.ty);
             out.has_graphics |= member.ty == "XrSession" || member.ty == "XrSwapchain";
             out.has_array |= member.static_array_len.is_some();
-            out.has_non_default |= member.ty == "XrTime" || member.ty == "XrDuration";
```

Comment:
Did you forget to add the `impl Default` in this commit?

Reply:
It was actually there already

Reply:
Sorry, I must have mixed it up with something else. I have added a derive for it now.